openapi-validator/src/main/java/io/ballerina/openapi/validator/ParameterExampleAnalyzer.java

```diff
@@ -0,0 +1,76 @@
+package io.ballerina.openapi.validator;
+
+import io.ballerina.compiler.api.SemanticModel;
+import io.ballerina.compiler.api.symbols.AnnotationAttachmentSymbol;
+import io.ballerina.compiler.api.symbols.ParameterSymbol;
+import io.ballerina.compiler.api.symbols.ResourceMethodSymbol;
+import io.ballerina.compiler.api.symbols.ServiceDeclarationSymbol;
+import io.ballerina.compiler.api.symbols.Symbol;
+import io.ballerina.compiler.api.symbols.TypeSymbol;
+import io.ballerina.compiler.api.symbols.resourcepath.PathRestParam;
+import io.ballerina.compiler.api.symbols.resourcepath.PathSegmentList;
+import io.ballerina.compiler.api.symbols.resourcepath.ResourcePath;
+import io.ballerina.compiler.syntax.tree.Node;
+import io.ballerina.projects.plugins.SyntaxNodeAnalysisContext;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Ballerina OpenAPI example annotation(s) analyzer for parameters.
+ *
+ * @since 2.1.0
+ */
+public class ParameterExampleAnalyzer extends AbstractExampleAnalyzer {
```

Comment:
Shall we move example-related validation into a separate package inside the validator package?

Reply:
Moved via [99b2782](https://github.com/ballerina-platform/openapi-tools/pull/1740/commits/99b2782dabd4f5949ed6e2d801983d605597bfa1)